dispatcher/MongoConnect.py

```diff
@@ -451,7 +513,7 @@ def send_command(self, command, hosts, user, detector, mode="", delay=0, force=F
                 "user": user,
                 "detector": detector,
                 "mode": mode,
-                "createdAt": datetime.datetime.utcnow()
+                "createdAt": now()
             }
             if command == 'arm':
                 doc_base['options_override'] = {'number': number}
```

Comment:
A few below, it reads: `docs['host'] = hosts[0]+hosts[1] if isinstance(hosts, tuple) else hosts`, how does that work?

Reply:
I really need some more comments in the lines below as they look quite hardcoded and I cannot fully judge if they will work correctly with the longer super-detector configs:
```python
            if delay == 0:
                docs = doc_base
                docs['host'] = hosts[0]+hosts[1] if isinstance(hosts, tuple) else hosts
                docs['acknowledged'] = {h:0 for h in docs['host']}
            else:
                docs = [dict(doc_base.items()), dict(doc_base.items())]
                docs[0]['host'], docs[1]['host'] = hosts
                docs[0]['acknowledged'] = {h:0 for h in docs[0]['host']}
                docs[1]['acknowledged'] = {h:0 for h in docs[1]['host']}
                docs[1]['createdAt'] += datetime.timedelta(seconds=delay)
            self.collections['command_queue'].insert(docs)
```

Reply:
`hosts` is a tuple of lists of CCs and readers. If we are issuing a command with a delay we have to make two separate documents; if we aren't then we just combine them.